src/cgi.h

```diff
@@ -49,8 +49,8 @@ extern int cgi_display_errors;
 extern void cgi_init_headers(void);
 extern void cgi_redirect(char *url);
 extern void cgi_fatal(const char *error);
```

Comment:
Quite right. Would you recommend going ahead with the changes or  reverting to not using 'const' for the time being?